starlette/debug.py

```diff
@@ -18,22 +21,22 @@ def get_debug_response(request, exc):
 
 
 class DebugMiddleware:
-    def __init__(self, app):
+    def __init__(self, app: ASGIApp) -> None:
         self.app = app
 
-    def __call__(self, scope):
+    def __call__(self, scope: Scope) -> typing.Union["_DebugResponder", ASGIInstance]:
```

Comment:
This oughta just be `-> ASGIInstance` (Since instances of _DebugResponder fulfil that interface), tho I don't know if that's problematic for mypy.

Reply:
Yes!!. There is no problem with `mypy`, it was my mistake!!.